src/taipy/core/config/core_section.py

```diff
@@ -93,6 +99,11 @@ def __init__(
         self.mode = mode or self._DEFAULT_MODE
         self.version_number = version_number or self._DEFAULT_VERSION_NUMBER
         self.force = force or self._DEFAULT_FORCE
+
+        self._core_version = core_version or self._DEFAULT_LATEST_CORE_VERSION
+        if self._core_version != self._DEFAULT_LATEST_CORE_VERSION:
```

Comment:
how do you handle minor or patched versions ie 3.0.1 ... ?

Reply:
I suppose there's a package for that ....
ex: https://pypi.org/project/sem-ver/

Reply:
this is still a topic I'm trying to improve with no good idea as of the moment

Reply:
Have decided to use the __version__ value in src/taipy/core/__init__.py